### PR TITLE
Task Plan Styles and Interactions

### PR DESCRIPTION
**Pull request by jivewise**
screen shot 2015-05-11 at 8 45 34 am
screen shot 2015-05-11 at 8 45 16 am
screen shot 2015-05-11 at 8 45 07 am

- [x] Fixed select reading accordion
- [x] Select Reading Dialog styles
- [x] Exercise Summary styles
- [x] Close buttons on homework and reading plans

### CODE REVIEW

File: resources/styles/components/task-step/reading.less [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+  .section-opener ~ section h1 + p::first-letter, div[data-type="document-title"] ~ p::first-letter {
+    float: left;
+    #fonts > .sans(11rem, 9rem);
+    font-weight: 600;
+    color: @physics-color-primary;
+    margin: 0 4px 0 -6px;
+  }
+
+  .reading-step {
```
Comment (philschatz) [inline resources/styles/components/task-step/reading.less]: Isn't this selector already defined above?

Comment (jivewise) [inline resources/styles/components/task-step/reading.less]: Not sure, I just moved this into task-step/ from task-plan/, didn't really go over the CSS, but I'll check it.

**Comment by philschatz** [comment]
Does this depend on any backend PR's?

**Comment by jivewise** [comment]
No, this is just front end changes for now.

**Pull request merged**